lib/main_common.pm

```diff
@@ -992,14 +991,18 @@ sub load_inst_tests {
         {
             # With SLE15 we change desktop using role and not by unselecting packages (Use SYSTEM_ROLE variable),
             # If we have minimalx, as there is no such a role, there we use old approach
-            loadtest "installation/installation_overview_before";
+            loadtest "installation/resolve_dependency_issues";
             loadtest "installation/change_desktop";
         }
     }
     if (get_var("UEFI") && get_var("SECUREBOOT")) {
         loadtest "installation/secure_boot";
     }
     if (installyaststep_is_applicable()) {
+        # Do not repeat step when change_desktop has been executed
+        if (!check_var("DESKTOP", "textmode")) {
+            loadtest "installation/resolve_dependency_issues";
```

Comment:
I'm not sure about this condition, you have removed handling from `change_desktop`.

Reply:
this is just to avoid repetition. instead to introduce a new variable i use this check with seems to be intuitive for the case as the change_desktop will trigger the installation/resolve_dependency_issues when DESKTOP=='textmode'. i have run some test but i could run more to verify that this doesnt cause any problem

Reply:
Could you please clarify the change.

It is not clear for me why there is the check for textmode desktop. As per my understanding, 'change_desktop' module might be called in a Test Suite with any 'DESKTOP' parameter. 

I guess the scheduling may be left the same as it was for 'installation_overview_before' module. Or I missed something?